src/auth/baseexternalclient.ts

```diff
@@ -57,6 +57,9 @@ export const EXTERNAL_ACCOUNT_TYPE = 'external_account';
 /** Cloud resource manager URL used to retrieve project information. */
 export const CLOUD_RESOURCE_MANAGER =
   'https://cloudresourcemanager.googleapis.com/v1/projects/';
+/** The workforce audience pattern. */
+const WORKFORCE_AUDIENCE_PATTERN =
+  '//iam.googleapis.com/locations/[^/]+/workforcePools/.+/providers/.+';
```

Comment:
`'//iam.googleapis.com/locations/[^/]+/workforcePools/[^/]+/providers/.+'`

You can refine it further as the pool ID cannot contain slashes